src/java-interop/java-interop-jvm.c

```diff
@@ -6,8 +6,8 @@
 #include "java-interop-util.h"
 
 
-typedef int (*java_interop_JNI_CreateJavaVM_fptr) (JavaVM **p_vm, void **p_env, void *vm_args);
-typedef int (*java_interop_JNI_GetCreatedJavaVMs_fptr) (JavaVM **vmBuf, int bufLen, int *nVMs);
+typedef int (JNICALL *java_interop_JNI_CreateJavaVM_fptr) (JavaVM **p_vm, void **p_env, void *vm_args);
```

Comment:
This looks important for part of https://github.com/xamarin/xamarin-android/pull/2525 ...

Reply:
The changes to this file are the *only* changes that appear to be required.